interim-jan-2021/resolver-behavior-self-description.md

```diff
@@ -0,0 +1,30 @@
+# Resolver behavior self-description
+
+## Defining local-only namespaces
+* Express namespaces which only this resolver can resolve
+    * Authoritatively if the namespace collides with any global names
+* Ex 1: Enterprise resolvers serving corporation-specific namespaces
+* Ex 2: Public Wi-Fi or cellular networks offering network-local services
+
+## Defining per-namespace optimization
+* Express namespaces for which this resolver provides preferable resolutions
+* Ex 1: ISP routes public content requests to network caches
+* Ex 2: Public resolver designated to serve a namespace to limit parties privy to resolution
+
+## Defining resolver identity
```

Comment:
Speaking of identity, I think this needs to convey the resolver's human-readable name.  I'd also like a favicon :-)

Reply:
I was about to heckle you about this, but... it would be nice to see something other than an IP address or URI when looking at DNS client configuration UI.

Adding a line.